xla/backends/gpu/codegen/triton/support.cc

```diff
@@ -63,7 +63,11 @@ bool IsTritonSupportedDataType(PrimitiveType type,
       return true;
     case F8E5M2:
     case F8E4M3FN:
-      return std::holds_alternative<se::CudaComputeCapability>(gpu_version);
+      return std::holds_alternative<se::CudaComputeCapability>(gpu_version) ||
```

Comment:
This is just equivalent to `return true` for now. Please fold these cases with the ones above.